src/app/views/query/form/query-form.component.js

```diff
@@ -210,18 +242,8 @@ class QueryFormController {
     this.params.bounds = null;
   }
 
-  filterExpeditions(query) {
-    return this.expeditions.filter(
-      e =>
-        !this.params.expeditions.includes(e) &&
-        (!query ||
-          e.expeditionTitle.toLowerCase().includes(query.toLowerCase())),
-    );
-  }
-
   generateFilterOptions() {
-    const { config } = this.currentProject;
-    this.filterOptions = config.entities.reduce(
+    this.filterOptions = this.params.config.entities.reduce(
```

Comment:
ah, can't you just access `this.config`, then you can remove from the `params` object